src/screens/Guest/Register.js

```diff
@@ -1,12 +1,31 @@
viewBox="0 0 1440 500" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient id="gradient" x1="0%" y1="50%" x2="100%" y2="50%"><stop offset="5%" stop-color="#002bdc66"></stop><stop offset="95%" stop-color="#32ded466"></stop></linearGradient></defs><path d="M 0,500 C 0,500 0,125 0,125 C 87.71006526966679,107.43765029199588 175.42013053933357,89.87530058399177 231,102 C 286.57986946066643,114.12469941600823 310.0295431123326,155.93644795602884 381,163 C 451.9704568876674,170.06355204397116 570.4616970113362,142.3789075918928 655,135 C 739.5383029886638,127.62109240810717 790.1236688423222,140.54792167639985 845,149 C 899.8763311576778,157.45207832360015 959.0436276193748,161.42940570250775 1017,151 C 1074.9563723806252,140.57059429749225 1131.7018206801786,115.73445551356922 1202,109 C 1272.2981793198214,102.26554448643078 1356.1490896599107,113.63277224321538 1440,125 C 1440,125 1440,500 1440,500 Z" stroke="none" stroke-width="0" fill="url(#gradient)" class="transition-all duration-300 ease-in-out delay-150 path-0"></path><defs><linearGradient id="gradient" x1="0%" y1="50%" x2="100%" y2="50%"><stop offset="5%" stop-color="#002bdc88"></stop><stop offset="95%" stop-color="#32ded488"></stop></linearGradient></defs><path d="M 0,500 C 0,500 0,250 0,250 C 72.86224665063554,235.76915149433182 145.72449330127108,221.53830298866367 214,212 C 282.2755066987289,202.46169701133633 345.96427344555127,197.6159395396771 416,215 C 486.03572655444873,232.3840604603229 562.4184129165236,271.99793885262795 628,279 C 693.5815870834764,286.00206114737205 748.3620748883545,260.39230504981106 817,255 C 885.6379251116455,249.60769495018891 968.1332875300586,264.4328409481278 1036,274 C 1103.8667124699414,283.5671590518722 1157.1047749914117,287.8763311576778 1222,283 C 1286.8952250085883,278.1236688423222 1363.4476125042943,264.0618344211611 1440,250 C 1440,250 1440,500 1440,500 Z" stroke="none" stroke-width="0" fill="url(#gradient)" class="transition-all duration-300 ease-in-out delay-150 path-1"></path><defs><linearGradient id="gradient" x1="0%" y1="50%" x2="100%" y2="50%"><stop offset="5%" stop-color="#002bdcff"></stop><stop offset="95%" stop-color="#32ded4ff"></stop></linearGradient></defs><path d="M 0,500 C 0,500 0,375 0,375 C 90.36551013397457,383.93369975953283 180.73102026794913,392.86739951906566 241,395 C 301.26897973205087,397.13260048093434 331.4414290621779,392.4641016832703 389,379 C 446.5585709378221,365.5358983167297 531.503263483339,343.27619374785303 604,351 C 676.496736516661,358.72380625214697 736.5455170044659,396.43112332531774 806,406 C 875.4544829955341,415.56887667468226 954.3146684987978,396.999312950876 1026,383 C 1097.6853315012022,369.000687049124 1162.1958090003434,359.57162487117824 1230,359 C 1297.8041909996566,358.42837512882176 1368.9020954998282,366.71418756441085 1440,375 C 1440,375 1440,500 1440,500 Z" stroke="none" stroke-width="0" fill="url(#gradient)" class="transition-all duration-300 ease-in-out delay-150 path-2"></path></SVG>
```

Comment:
SVG açık şekilde yazılmaktansa komponent haline getirilebilir.

Reply:
O zaman her farklı svg için ayrı component oluşturmak gerekecek. Ama yapılabilir. Ama üzerine issue açıp düzeltelim.